networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -38,8 +38,9 @@ type QuestionTemplate @model {
   title: String!
   owner: String!
   version: Int!
-  choices: String
-  instructions: String
+  choices: AWSJSON
+  instructions: AWSJSON
+  answerSettings: AWSJSON
```

Comment:
Can this be nullable? Should we have a default value for it?

Reply:
I've set values in the `graphql.schema` file and they are also initialized in `createQuestion`